web/auth0-debug/htdocs/auth_config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "dev-bu-hbv3o.us.auth0.com",
+  "clientId": "k3nFXxrQFVinXsGGpdT4dKAtfKYZbQle",
+  "audience": "https://bff.vaspdirectory.net"
+}
```

Comment:
This is all public info that would be configured in a single page app (SPA) front-end.